src/isEmpty.js

```diff
@@ -12,10 +12,13 @@ var _curry1 = require('./internal/_curry1');
  * @return {Boolean}
  * @example
  *
- *      R.isEmpty([1, 2, 3]); //=> false
- *      R.isEmpty([]); //=> true
- *      R.isEmpty(''); //=> true
- *      R.isEmpty(null); //=> false
+ *      R.isEmpty([1, 2, 3]);   //=> false
+ *      R.isEmpty([]);          //=> true
+ *      R.isEmpty('');          //=> true
+ *      R.isEmpty(null);        //=> false
+ *      R.isEmpty(R.keys({}));  //=> true
+ *      R.isEmpty({});          //=> false ({} does not have a length property)
+ *      R.isEmpty({length: 0}); //=> true
```

Comment:
Would we then state the following?

```javascript
R.isEmpty('');          //=> true ('' is array-like)
```

I don't think of strings as array-like.

For the record I remain in favour of removing this function. It's just sugar for `R.propEq('length', 0)`, but this fact is not obvious.